openapi-v1.yaml

```diff
@@ -2632,6 +2632,95 @@ definitions:
       - array:read-write
       - array:admin
 
+  GroupAttributes:
+    description: Attributes that describe the group itself, not any of the subgroups or assets
+    type: object
+    properties:
+      id:
+        description: the globally unique id of the group
+        type: string
+        x-omitempty: true
+      file_type:
```

Comment:
Why is this at the "Group" level? If the group can have multiple types in it what does this do in those cases? I don't believe this is needed.

Reply:
True it is not needed. It is provided for UI purposes. If a group contains an array and a notebook under which UI tab, arrays, notebooks etc should this appear? For the first iteration we cannot support easily both